src/comp/pages/UserInvitePage.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+
+import UserInputField from "comp/UserInputField";
+import * as style from "css/comp/pages/UserInvitePage.module.css";
+import * as CommonStyle from "css/comp/Common.module.css";
+import CreateSendButton from "../SendButton";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "@material-ui/core";
+import { RouteComponentProps } from "react-router-dom";
+
+type UserInvitePageState = {
+  selectedUserId: string[];
+  isConfirmDialogOpen: boolean;
```

Comment:
```suggestion
  openedConfirmDialog: boolean;
```
ちょっとわかりにくかったので、あくまで提案ですが